lib/was_crawl_preassembly/utilities.rb

```diff
@@ -2,9 +2,8 @@ module Dor
   module WASCrawl
     class Utilities
       def self.get_collection_id(druid_obj)
-        collections = druid_obj.collections
-        raise "#{druid_obj.id} doesn't belong to a collection" if collections.length == 0
-        collection = collections[0]
+        collection = druid_obj.collections.first
```

Comment:
Yes, if `druid_obj` is nil (or something other than ActiveFedora::Base) then the caller is wrong and I didn't check for that. We can add `fail unless druid_obj.respond_to?(:collections)` if we want to do type checking.

`druid_obj.collections` is an ActiveFedora relation. It's not a variable per se -- it's defined by an ActiveFedora method `has_many_and_belongs_to` -- see https://github.com/sul-dlss/dor-services/blob/master/lib/dor/models/concerns/governable.rb#L7

The `.first == nil` is based on their example here: https://github.com/projecthydra/active_fedora/blob/v8.3.0/lib/active_fedora/associations/collection_proxy.rb#L172. I've also seen `.empty?` but we actually want the data anyway. That said, I think `druid_obj.collections.select(:id).first` would also work and might be better for clarity -- we only want the collection.id value in this method. Their example is here https://github.com/projecthydra/active_fedora/blob/v8.3.0/lib/active_fedora/associations/collection_proxy.rb#L95